manifestservice/manifests/__init__.py

```diff
@@ -122,6 +124,78 @@ def put_manifest():
 
     return flask.jsonify(ret), 200
 
+@blueprint.route("/cohorts", methods=["GET"])
```

Comment:
i see this service doesn't have a swagger doc :( could you update the readme to add these endpoints 🙏